### PR TITLE
Increase timeout when downloading dependencies

### PR DESCRIPTION
**Pull request by teetone**
### Reasons for making this change

When a worker downloads a large dependency, it will exceed the read timeout. This PR increases the default timeout and adds a separate timeout for downloading the bundle contents.

### Checklist

* [x] I've added a screenshot of the changes, if this is a frontend change
* [x] I've added and/or updated tests, if this is a backend change
* [x] I've run the [pre-commit.sh](https://github.com/codalab/codalab-worksheets/blob/master/pre-commit.sh) script
* [x] I've updated [docs](https://github.com/codalab/codalab-worksheets/tree/master/docs), if needed

### CODE REVIEW

**Review by yiboliu — APPROVED** [review]

**Comment by yiboliu** [comment]
As long as our customers are happy with this significant increase in timeout

**Comment by nelson-liu** [comment]
this seems fine---there used to not even be a timeout, so i think just having one is the right thing.

**Review by nelson-liu — APPROVED** [review]

**Comment by nelson-liu** [comment]
hm looks like 1d63f7a3c6c6656fab9b01912bf6cd9d06c0a0bf introduces some format violations that need to be fixed before this can be merged.

File: codalab/worker/bundle_service_client.py [hunk 1]
```diff
@@ -179,5 +179,6 @@ def get_bundle_contents(self, uuid, path):
             '/bundles/' + uuid + '/contents/blob/' + path,
             headers={'Accept-Encoding': 'gzip'},
             return_response=True,
+            timeout_seconds=10 * 60,
```
Comment (epicfaace) [inline codalab/worker/bundle_service_client.py]: Should this be set to `URLOPEN_TIMEOUT_SECONDS`?

Comment (nelson-liu) [reply codalab/worker/bundle_service_client.py]: I think the intention was to make the timeout a bit more lenient for `/bundles/contents/blob/`

Comment (nelson-liu) [reply codalab/worker/bundle_service_client.py]: Does this timeout mean that the download has to take less than `10*60` seconds?

Comment (epicfaace) [reply codalab/worker/bundle_service_client.py]: Perhaps we set it to `URLOPEN_TIMEOUT_SECONDS * 2` instead then?

Comment (teetone) [reply codalab/worker/bundle_service_client.py]: Updated

**Pull request merged**